### DISCUSSION TITLE
24. January 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1133] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] Benny: Current status of docu restructuring

**Comment by woju** [comment]
- [ ] packages for RHEL 9 and derivatives — do we release them and if so, how officially?

**Comment by woju** [comment]
Slide from followup on security process:
![gramine-security-process-00](https://user-images.githubusercontent.com/1550337/214343718-7a9d0e3d-bf3e-4b5a-a026-21b7254a7b69.png)
